operations/observability/mixins/IDE/rules/vscode.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+# Licensed under the GNU Affero General Public License (AGPL).
+# See License-AGPL.txt in the project root for license information.
+
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: openvsx-proxy-monitoring-rules
+  namespace: monitoring-satellite
+spec:
+  groups:
+  - name: vscode
+    rules:
+    - alert: VSCodeExtensionInstallFailuresRatioTooHigh
+      labels:
+        severity: critical
+      for: 20m
+      annotations:
+        runbook_url: https://github.com/gitpod-io/runbooks/blob/main/runbooks/VSCodeExtensionInstallFailuresRatioTooHigh.md
+        summary: Open-VSX registry is possibly down
+        description: Open-VSX registry or our openvsx-mirror is possibly down. Some user cannot install VSCode extensions.
+        dashboard_url: https://grafana.gitpod.io/d/oLzOteZ4z/vs-code-browser-overview
+      expr: |
```

Comment:
looking at i guess it would be alright > 0.2
<img width="1904" alt="Screenshot 2022-11-11 at 16 27 58" src="https://user-images.githubusercontent.com/3082655/201371961-2853f7a5-4e7b-45dd-8d3d-e510b83f47e2.png">

Reply:
although there was a continuous spike without mirror before as well
<img width="1904" alt="Screenshot 2022-11-11 at 16 28 59" src="https://user-images.githubusercontent.com/3082655/201372176-ba92825a-4dbd-44da-89ca-695da7dfbc06.png">

maybe 0.25 is better